java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSink.java

```diff
@@ -92,8 +94,23 @@ private PreparedStatement prepareStatement(SinkRow row) {
                 try {
                     PreparedStatement stmt =
                             conn.prepareStatement(insertStmt, Statement.RETURN_GENERATED_KEYS);
+                    var columnNames = getTableSchema().getColumnNames();
                     for (int i = 0; i < row.size(); i++) {
-                        stmt.setObject(i + 1, row.get(i));
+                        switch (getTableSchema().getColumnType(columnNames[i])) {
+                            case INTERVAL:
+                                stmt.setObject(i + 1, new PGInterval((String) row.get(i)));
```

Comment:
Since the underlying data type for `INTERVAL` and `JSONB` is String when we call `row.get(i)`, the support of these two data types on json payload should be simple. Shall we added support also on json payload, and then we can write our unit test to test whether the two data types work with bost postgreSQL and MySQL.

Reply:
Yeah, seems it is simple to add data types if they have string representation. I'll add them to json payload for the PG sink too.